packages/pigeon/example/README.md

```diff
@@ -31,6 +31,7 @@ needed for your project.
   // Set this to a unique prefix for your plugin or application, per Objective-C naming conventions.
   objcOptions: ObjcOptions(prefix: 'PGN'),
   copyrightHeader: 'pigeons/copyright.txt',
+  packageName: 'pigeonExamplePackage',
```

Comment:
`pigeon_example_package`; since the default is a package name, we should follow package naming conventions for consistency.